README.md

```diff
@@ -568,6 +627,6 @@ If you've made it all the way to the end here, then you've learned that you can
 
 There's certainly more you can do with Drupal 8's Migrate API.  There's a plethora of source and processing plugins out there that can handle pretty much anything you throw at it.  XML and JSON are fair game.  You can also request sources using HTTP, so you can always point it at an existing systems REST API and go from there.  Eventually, you'll also have to write your own process plugin if you can't make your logic work with the Migrate API's sometimes awkward workflow.
 
-But really the best thing to do is try and get your data into Islandora!  You can use the `boilerplate` branch of this repository to clone down an empty migration, ready for you to customize to fit your data.  And as you assmble it into CSV format, keep in mind that if you have more than just names for things like subjects and authors, that you can always make more CSVs.  Think of it like maintaining tables in an SQL database.  Each CSV has unique keys, so you can lookup/join entiities between CSVs using those keys.  And you can still pipe delimit the keys like we did in our example to handle multi-vlaued fields.
+But really the best thing to do is try and get your data into Islandora!  We intend to create a `boilerplate` branch of this repository that will allow you to clone down a migration template, ready for you to customize to fit your data.  And as you assemble it into CSV format, keep in mind that if you have more than just names for things like subjects and authors, that you can always make more CSVs.  Think of it like maintaining tables in an SQL database.  Each CSV has unique keys, so you can lookup/join entiities between CSVs using those keys.  And you can still pipe-delimit the keys like we did in our example to handle multi-valued fields.
```

Comment:
yeah I don't see a `boilerplate` branch yet! I figure we shouldn't lie until we have one shipped.

Reply:
done!